cumulusci/core/dependencies/tests/test_resolvers.py

```diff
@@ -132,6 +157,30 @@ def test_github_release_tag_resolver(self, project_config):
             ),
         )
 
+    def test_github_release_tag_resolver__2gp(self, project_config):
```

Comment:
It looks like this is a duplicate of the test above (with a different repo name).

Edit: Ah, it looks like we are testing different repos here (that are defined in `conftest.py`), but this test and the one above it need to have unique names (they currently share the same one).

Reply:
Fixed!